lib/ansible/modules/package.py

```diff
@@ -40,6 +40,8 @@
     description:
       - The required package manager module to use (V(dnf), V(apt), and so on). The default V(auto) will use existing facts or try to autodetect it.
       - You should only use this field if the automatic selection is not working for some reason.
+      - Since version 2.17 you can override the automatic detection both through a local facts C(overrides) file and setting
```

Comment:
We normally add it to the prose if it modifies an existing option so users reading the 'latest' do not confuse it with it working on older versions. If this added a new option with `version_added`, it would not be in the prose.